app_spec/test/test.js

```diff
@@ -17,6 +17,48 @@ const scenario1 = new Scenario([instanceAlice], { debugLog:true })
 const scenario2 = new Scenario([instanceAlice, instanceBob], { debugLog: true })
 const scenario3 = new Scenario([instanceAlice, instanceBob, instanceCarol], { debugLog: true })
 
+scenario2.runTape('capabilities grant and claim', async (t, { alice, bob }) => {
+    /*
+      This is not a perfect test of requesting a grant because currently there
+      is no way in the test conductor to actually pass in the provenance of the
+      call.  That will be added when we convert the test framework to being built
+      on top of the rust conductor.
```

Comment:
Is this still the case given the other comment about directly specifying an alternate provenance?

Reply:
good point deleted